crypto/fipsmodule/aes/mode_wrappers.c

```diff
@@ -57,23 +57,16 @@
 // hwaes_capable when on in ARM uses 10, 12, 14 for key rounds.
```

Comment:
Is this what you meant to say in the comment?
```suggestion
// hwaes_capable when not on ARM uses 10, 12, 14 for key rounds.
```

Reply:
reclarified 👍